beta/nncf/tensorflow/quantization/algorithm.py

```diff
@@ -157,9 +209,29 @@ def get_transformation_layout(self, model):
 
         return transformations
 
+    def build_controller(self, model: tf.keras.Model) -> 'QuantizationController':
+        return QuantizationController(model, self.config)
+
+    def initialize(self, model: tf.keras.Model) -> None:
+        self._run_range_initialization(model)
+
+        if self._batchnorm_adaptation:
+            self._run_batchnorm_adaptation(model)
+
+    def _run_range_initialization(self, model: tf.keras.Model) -> None:
+        if self._range_initializer is None:
+            self._range_initializer = MinMaxInitializer(self._range_init_params)
```

Comment:
I suggest implementing the `extract_range_init_params()` method and using it here by the same way as the `extract_bn_adaptation_init_params()` method is used in the `_run_batchnorm_adaptation()` method. I think it's a very clear design.

Reply:
There are many hyperparameters in the range initialization algorithm. I think the question of which API the quantization range initialization algorithm should have is an open question. @negvet, keep in mind this comment when you will introduce the initialization algorithm.